app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :post, foreign_key: 'author_id'
+  has_many :comment, foreign_key: 'author_id'
+  has_many :like, foreign_key: 'author_id'
```

Comment:
- [ ] Please notice that when setting up a has_many relation the name of the other model must be pluralised as mentioned in the [rails guides](https://guides.rubyonrails.org/association_basics.html#the-has-many-association) **"The name of the other model is pluralized when declaring a has_many association."**
Please fix this all over your codebase, wherever you have used singular name for setting up a has_many relation :+1:

Reply:
- Thanks this has been updated.